Mathlib/FieldTheory/SeparableDegree.lean

```diff
@@ -130,13 +130,15 @@ variable (K : Type w) [Field K] [Algebra F K]
 namespace Field
 
 /-- `Field.Emb F E` is the type of `F`-algebra homomorphisms from `E` to the algebraic closure
-of `E`. -/
+of `E`.
+-/
 def Emb := E →ₐ[F] AlgebraicClosure E
 
 /-- If `E / F` is an algebraic extension, then the (finite) separable degree of `E / F`
 is the number of `F`-algebra homomorphisms from `E` to the algebraic closure of `E`,
 as a natural number. It is defined to be zero if there are infinitely many of them.
-Note that if `E / F` is not algebraic, then this definition makes no mathematical sense. -/
+Note that if `E / F` is not algebraic, then this definition makes no mathematical sense.
+-/
```

Comment:
I'm not sure if this is preferred.

Reply:
I'll follow the original author's style at the beginning. If the author uses 
```
/-- foo
```
then it should be
```
bar. -/
```
If the author uses
```
/--
foo
```
then it should be
```
bar.
-/
```

Reply:
The inconsistency here is caused by we originally wrote Stacks links in the comments. When we changed to the stacks attribute, we didn't delete the line breaks correctly.